udata/core/user/search.py

```diff
@@ -49,8 +49,8 @@ class UserSearch(ModelSearchAdapter):
         'views': Sort('metrics.views'),
         'created': Sort('created'),
     }
-    facets = {
-        'organization': ModelTermFacet('organizations', Organization),
+    aggregations = {
+        'user_organizations': ModelTermFacet('user_organizations', Organization),
```

Comment:
@noirbizarre why singular? We're retrieving many organizations with that facet, it looks more consistent to me like that.